public/css/gogs.css

```diff
@@ -2691,3 +2691,28 @@ footer .ui.language .menu {
 .ui.user.list .item .description a:hover {
   text-decoration: underline;
 }
+.markdown-body table {
```

Comment:
You should never change `.css` files manually?

And how does this effect regular HTML tables?